test/widgets/actions_test.dart

```diff
@@ -269,4 +270,37 @@ void main() {
         expectedMessage: 'NetworkException: Oops (ClientException: Oops)');
     }, skip: true);
   });
+
+  group('updateMessageFlagsStartingFromAnchor', () {
+    String onCompletedMessage(int count) => 'onCompletedMessage($count)';
+    const progressMessage = 'progressMessage';
+    const onFailedTitle = 'onFailedTitle';
+
+    testWidgets('smoke test', (tester) async {
+      final narrow = TopicNarrow.ofMessage(eg.streamMessage());
+      final apiNarrow = narrow.apiEncode()..add(ApiNarrowIsUnread());
+      await prepare(tester);
+      connection.prepare(json: UpdateMessageFlagsForNarrowResult(
+        processedCount: 11, updatedCount: 3,
+        firstProcessedId: null, lastProcessedId: null,
```

Comment:
Let's try to use realistic values for `firstProcessId` and `lastProcessedId`, to be consistent with what the server might return.